src/website/views/custom-elements/button.pug

```diff
@@ -0,0 +1,268 @@
+---
+title: Button
+scripts:
+  - js/ce/ux-chi-ce
+
+---
+p.-text
+  | Buttons are used to trigger actions in forms, modals, and more.
+
+h3 Button tags
```

Comment:
@srinath-koneru in CE we are not supporting different tags. In fact, all the buttons render a button, so the doc with link, input... is not necessary. Remove these examples and just leave Button example.

Reply:
@srinath-koneru this comment has not been updated.

Reply:
@jllr  just resolved it.